src/models/inbound/dfspLinking.interface.ts

```diff
@@ -81,6 +81,12 @@ export interface BackendGetScopesResponse {
   scopes: tpAPI.Schemas.Scope[]
 }
 
+export interface BackendStoreValidatedConsentRequest {
```

Comment:
Is there a reason you're not also sending the credential along here?

Reply:
I can. Do you want me to just send the whole credential?

Reply:
Yeah I think so... that way the DFSP has it for auditing purposes or if they want to check the credential themselves.